docs/en/verifying-signatures.wml

```diff
@@ -387,24 +393,25 @@ Primary key fingerprint: EF6E 286D DA85 EA2A 4BA7  DE68 4E2C 6E87 9329 8290
     Notice that there is a warning because you haven't assigned a trust
     index to this person. This means that GnuPG verified that the key made
     that signature, but it's up to you to decide if that key really belongs
-    to the developer. The best method is to meet the developer in person and
-    exchange key fingerprints.
+    to the developer. As international travel to meet the developer might be
+    unfeasable you are left with trusting other people who signed this key.
```

Comment:
This explanation raises security concerns, but it doesn't tell people what they should do.

If you can't give people a useful action to take to improve their security, please just tell them to ignore the warning.

One useful action might be: "use another device or another internet connection to check the key fingerprints listed on the tor website at ..."